content/en/docs/collector/builder.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Builder
+weight: 25
+---
+
+The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
+generates sources and binaries for OpenTelemetry Collector
+[distributions](../distributions).
+
+This tool is currently being used to build both the core and the contrib
+distributions, and can be used to build custom distributions as well, allowing
+users to cherry-pick which components they need for their production usage,
+potentially even including their own custom components.
```

Comment:
```suggestion
This tool is currently being used to build both the [core](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol) and the [contrib](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib)
distributions. It can be used to build custom distributions as well, allowing
users to cherry-pick which components they need for their production usage,
potentially even including their own custom components.
```